README.md

```diff
@@ -227,6 +227,59 @@ If you want a PDB for such a deployment, you can either:
 
 This behavior applies to both integer values (`maxUnavailable: 1`) and percentage values (`maxUnavailable: 25%`). Only deployments with `maxUnavailable: 0` or `maxUnavailable: 0%` will automatically get PDBs created.
 
+### Enabling Eviction Autoscaler for Specific Namespaces
+
+By default, eviction autoscaler is **enabled** for all deployments in the `kube-system` namespace. For deployments in other namespaces, you must explicitly enable the eviction autoscaler by adding the following annotation **to the namespace**:
+
+```yaml
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: production
+  annotations:
+    eviction-autoscaler.azure.com/enable: "true"
```

Comment:
when I disable a namespae is it up to me to clear out pdbs? 

Maybe we should give them a simple kubelet command htat lets them select pdbs we created by a label and delete?